Windows build fails for 4.3.0-alpha.1

Windows builds appear to be broken.

```
$ make -C build.assets release-windows
[...]
---> Building OSS binaries.
/usr/bin/make build/tsh
GOOS=windows GOARCH=amd64 CGO_ENABLED=1 go build -tags "pam " -o build/tsh -ldflags -w -ldflags '-w -s' ./tool/tsh
# github.com/gravitational/teleport/lib/client
lib/client/api.go:1478:33: too many arguments in call to newSession
	have (*NodeClient, nil, map[string]string, io.Reader, io.Writer, io.Writer, bool, bool)
	want (*NodeClient, *session.Session, map[string]string, io.Reader, io.Writer, io.Writer, bool)
lib/client/api.go:1512:32: too many arguments in call to newSession
	have (*NodeClient, *session.Session, map[string]string, io.Reader, io.Writer, io.Writer, bool, bool)
	want (*NodeClient, *session.Session, map[string]string, io.Reader, io.Writer, io.Writer, bool)
lib/client/api.go:2269:33: cannot use fd (type syscall.Handle) as type int in argument to terminal.GetState
lib/client/api.go:2281:21: cannot use fd (type syscall.Handle) as type int in argument to terminal.Restore
lib/client/api.go:2291:37: cannot use fd (type syscall.Handle) as type int in argument to terminal.ReadPassword
make[2]: *** [build/tsh] Error 2
```

--- a/lib/client/api.go
+++ b/lib/client/api.go
@@ -2265,7 +2265,11 @@ func (tc *TeleportClient) getServerVersion(nodeClient *NodeClient) (string, erro
 
 // passwordFromConsole reads from stdin without echoing typed characters to stdout
 func passwordFromConsole() (string, error) {
-	fd := syscall.Stdin
+	// syscall.Stdin is not an int on windows. The linter will complain only on
+	// linux where syscall.Stdin is an int.
+	//
+	// nolint:unconvert
+	fd := int(syscall.Stdin)
 	state, err := terminal.GetState(fd)
 
 	// intercept Ctr+C and restore terminal

--- a/lib/client/session_windows.go
+++ b/lib/client/session_windows.go
@@ -35,13 +35,16 @@ type NodeSession struct {
 	ExitMsg string
 }
 
-func newSession(client *NodeClient,
+func newSession(
+	client *NodeClient,
 	joinSession *session.Session,
 	env map[string]string,
 	stdin io.Reader,
 	stdout io.Writer,
 	stderr io.Writer,
-	legacyID bool) (*NodeSession, error) {
+	legacyID bool,
+	enableEscapeSequences bool,
+) (*NodeSession, error) {
 
 	return nil, trace.BadParameter("sessions not supported on Windows")
 }

